HQSmokeTests/testCases/conftest.py

```diff
@@ -0,0 +1,154 @@
+import os
+from configparser import ConfigParser
+from pathlib import Path
+import pytest
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from HQSmokeTests.testPages.loginPage import LoginPage
+from selenium.webdriver.chrome.options import Options
+from HQSmokeTests.UserInputs.userInputsData import UserInputsData
+from HQSmokeTests.utilities.email_pytest_report import Email_Pytest_Report
+
+driver = None
+
+def load_settings_from_environment():
+    """Load settings from os.environ
+
+            Names of environment variables:
+                DIMAGIQA_URL
+                DIMAGIQA_LOGIN_USERNAME
+                DIMAGIQA_LOGIN_PASSWORD
+
+            See https://docs.github.com/en/actions/reference/encrypted-secrets
+            for instructions on how to set them.
+            """
+    settings = {}
+    for name in ["url", "login_username", "login_password"]:
+        var = f"DIMAGIQA_{name.upper()}"
+        if var in os.environ:
+            settings[name] = os.environ[var]
+    return settings
+
+
+def load_settings():
+    if os.environ.get("CI") == "true":
+        settings = load_settings_from_environment()
+        if not settings:
+            raise RuntimeError(
+                f"Environment variables not set:\n"
+                "  DIMAGIQA_URL\n"
+                "  DIMAGIQA_LOGIN_USERNAME\n"
+                "  DIMAGIQA_LOGIN_PASSWORD\n\n"
+                "See https://docs.github.com/en/actions/reference/encrypted-secrets "
+                "for instructions on how to set them."
+            )
+        return settings
+    path = Path(__file__).parent.parent / "settings.cfg"
+    if not path.exists():
+        raise RuntimeError(
+            f"Not found: {path}\n\n"
+            "Copy settings-sample.cfg to settings.cfg and populate "
+            "it with values for the environment you want to test."
+        )
+    settings = ConfigParser()
+    settings.read(path)
+    return settings["default"]
+
+@pytest.fixture(scope="class")
+def init_driver(request):
+    settings = load_settings()
+    chrome_options = Options()
+    global driver
```

Comment:
globals like this are an antipattern and given that you're using pytest fixtures you shouldn't need it

Reply:
Looks like you're using this for the reporting below. You can access the driver in the report function like this:
```python
request = item.funcargs['request']
driver = request.getfuncargvalue('driver')
```
https://stackoverflow.com/a/57606735/632517